public/js/stores/management.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+var actionTypes = require('constants/action-types');
+
+export default function management(state, action) {
+  console.log('management store: got action', action);
+
+  if (action.type === actionTypes.APP_ERROR) {
+    return {
+      loading: false,
+      error: {
+        debugMessage: action.error.debugMessage,
+      },
```

Comment:
add `paymentMethods: null` ? You could also make a default state where each return statement clones that object.